packages/union-component/src/components/DonationFundDropdown.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { SelectHTMLAttributes, useEffect, useState } from 'react';
+import DonationDropdown from './DonationDropdown';
+import { getMembershipFunds, IFund } from './utils';
+
+interface Props extends SelectHTMLAttributes<HTMLSelectElement> {
+  setFundId: (fundId: string) => void;
+}
+
+const DonationFundDropdown: React.FC<Props> = (props) => {
+  const { setFundId } = props;
+  const [options, setOptions] = useState<IFund[]>([]);
+
+  useEffect(() => {
+    const getOptions = async () => {
+      const funds = await getMembershipFunds();
+
+      if (funds?.length) {
+        setOptions(funds);
+        setFundId(funds[0].id as string);
```

Comment:
if you are dealing with numbers why not change line 6 to: `setFundId: (fundId: number) => void;`?